.prettierrc.toml

```diff
@@ -0,0 +1,9 @@
+# See https://prettier.io/docs/en/options.html.
+# Prettier also reads .editorconfig.
+printWidth = 80
+singleQuote = true
+quoteProps = 'consistent'
+trailingComma = 'all'
+arrowParens = 'always'
+proseWrap = 'preserve'
+vueIndentScriptAndStyle = true
```

Comment:
I think we should replace this with Wagtail’s `.editorconfig` and `prettier.config.js`:

- https://github.com/wagtail/wagtail/blob/main/.editorconfig
- https://github.com/wagtail/wagtail/blob/main/prettier.config.js

Reply:
Thanks - added.